weave/trace/weave_client.py

```diff
@@ -859,13 +870,44 @@ def get_calls(
             columns: A list of columns to include in the response. If None,
                all columns are included. Specifying fewer columns may be more performant.
                Some columns are always included: id, project_id, trace_id, op_name, started_at
+            scored_by: A list of Scorers to filter by. Multiple scorers are ANDed together. You can
```

Comment:
It would be nice if you could also pass a single string here instead of a list with one item.
It would also be nice if instead of a full ref URL you could specify just `:v1` or the digest